src/protobuf/stmt.proto

```diff
@@ -23,6 +23,7 @@ message SimpleStatement {
   string object = 1;
   repeated string refs = 2;
   repeated string tags = 3;
+  repeated string deps = 4;
```

Comment:
Ah ok so the pb decoder creates an empty array for 0x repeated elements, so these types are wrong in types.js:

```js
export type SimpleStatementMsg = {
  object: string,
  refs?: Array<string>,
  tags?: Array<string>,
  deps?: Array<string>
};
```

Those should not be `?`